### ISSUE TITLE
Allowing to input just specific folders in config.json

### ISSUE THREAD

**Issue by lwcorp** [breakerh/imap-backup-dovecot#4]
Since most e-mail accounts are huge, this script is bound to get stuck not by its own merit.

Please consider allowing to input just specific folders inside `config.json` so the migration will be less massive and much less likely to get stuck. One could migrate just a folder or two, then other two, etc.

**Comment by breakerh** [comment]
Good idea! Will do :) will also add a function to list all directory's for that use case.
